content/spin/v2/key-value-store-tutorial.md

```diff
@@ -500,35 +517,13 @@ Interestingly there is one more request we can re-run before wrapping up this tu
 <!-- @selectiveCpy -->
 
 ```bash
-$ curl -I HEAD localhost:3000/test -v
+$ curl -I localhost:3000/test
 
-Trying 127.0.0.1:3000...
-Connected to localhost (127.0.0.1) port 3000
-HEAD /test HTTP/1.1
-Host: localhost:3000
 HTTP/1.1 404 Not Found
 ```
 
 As we can see above, there is currently no data found at the `/test` endpoint of our application.
 
 ## Conclusion
 
-We want to get feedback on the ergonomics of the key value API. We are curious about what new APIs you would suggest we implement and are also interested in learning about what backing stores you would like to see.
-
-## Next Steps
-
-You can read the [improvement proposal for key/value support](https://github.com/fermyon/spin/blob/main/docs/content/sips/010-key-value.md) as well as the implementation for the [current feature](https://github.com/fermyon/spin/pull/1035). Please feel free to ask questions and also share your thoughts in [our Discord community](https://discord.gg/AAFNfS7NGf).
-
-<script type="application/ld+json">
-{
-  "@context": "https://schema.org",
-  "@type": "VideoObject",
-  "name": "Key/Value Store - Spin 0.10.0",
-  "description": "Overview of the key/value store in Spin v0.10.0",
-  "thumbnailUrl": "https://www.fermyon.com/static/image/twc-spin.png",
-  "uploadDate": "2023-02-25T08:00:00+00:00",
-  "duration": "PT3M16S",
-  "contentUrl": "https://www.youtube.com/watch?v=qNBnVA2pkkY",
-  "embedUrl": "https://www.youtube.com/embed/qNBnVA2pkkY"
-}
-</script>
+We want to get feedback on the ergonomics of the key value API. We are curious about what new APIs you would suggest we implement and are also interested in learning about what backing stores you would like to see.
```

Comment:
I am not sure this paragraph is relevant at this stage.  Yes we are always happy for feedback but that is true of everything.  This API seems like it's past the "testing the water" stage.

Reply:
👍🏻 - Removed.